src/main/java/com/notnoop/apns/internal/SSLContextBuilder.java

```diff
@@ -0,0 +1,150 @@
+package com.notnoop.apns.internal;
```

Comment:
mind adding a license header here ? 

I think you can skipt this line in the header:
https://github.com/spark404/java-apns/blob/multiple-certs-in-store/src/main/java/com/notnoop/apns/ApnsServiceBuilder.java#L2

(or use your own name)